package.json

```diff
@@ -46,12 +46,13 @@
     ],
     "testPathIgnorePatterns": [
       "/node_modules/",
+      "<rootDir>/integration-tests/create-react-app",
       "<rootDir>/packages/react-percy-webpack/src/entry/__tests__/test-project/"
     ]
   },
   "lint-staged": {
     "*.js": [
-      "eslint --fix --color --no-ignore",
```

Comment:
I set ESLint to ignore the `create-react-app` project since it doesn't follow the Percy style conventions. Wanted to leave it as out-of-the-box as possible. However, this `--no-ignore` flag was added recently which tells ESLint to run even on files you've explicitly ignored when committing. Went ahead and removed it, not sure why it was added. @timhaines do you know?